jwst/datamodels/container.py

```diff
@@ -464,7 +468,7 @@ def get_crds_parameters(self):
         Notes
         -----
         stpipe requires ModelContainer to have a crds_observatory attribute in order
-        to pass through step.run(), but it is never accessed.
+        to pass through ``step.run()``, but it is never accessed.
```

Comment:
Similarly this note appears to be developer-facing.  It's also weird that this sits under `get_crds_parameters` instead of `crds_observatory` property